docs/sources/datasources/tempo/query-editor/_index.md

```diff
@@ -101,6 +101,10 @@ To query a particular trace:
 
 ## Query Loki for traces
 
+{{< admonition type="caution" >}}
+Starting with release Grafana v11, the Loki query tab will no longer be available.
+{{< /admonition >}}
+
 To find traces to visualize, you can use the [Loki query editor]({{< relref "../../loki#loki-query-editor" >}}).
 For results, you must configure [derived fields]({{< relref "../../loki#configure-derived-fields" >}}) in the Loki data source that point to this data source.
```

Comment:
I'd specify what kind of results. Dashboard query results?

Reply:
This content appears specifically on the query editor page, so it's only the query editor and not related to dashboards in any way. Also, the screenshot with the text shows the query editor. Do you think the context is enough or should we state something specifically?  

Here's the page for context: https://grafana.com/docs/grafana/latest/datasources/tempo/query-editor/#query-loki-for-traces